internal/test/deployment/pmm.go

```diff
@@ -171,7 +171,9 @@ func (p *PMM) ClickhouseURL() string {
 	if err != nil {
 		p.t.Fatal(err)
 	}
-	u.User = nil
+
+	u.User = url.UserPassword("default", "clickhouse")
```

Comment:
if `u.User == nil`?

Reply:
The user can be passed via the url as well.

Reply:
so turns out u.User starts out as admin (based on pmm URL), and we need to remove the nil check to force clickhouse credentials.

this is also only used for test (and it's why tests fail atm).